assets/bingo/lang/pl_pl.json

```
@@ -0,0 +1,411 @@
+
+	"bingo.lobby.tutorial.game_info.sign.line1": "",
+	"bingo.lobby.tutorial.game_info.sign.line2": "Informacje",
+	"bingo.lobby.tutorial.game_info.sign.line3": "o Grze",
+	"bingo.lobby.tutorial.game_info.sign.line4": "",
+
+	"bingo.lobby.tutorial.game_info.introduction.heading": "Informacje o Grze",
+	"bingo.lobby.tutorial.game_info.introduction": "Poniższa książka zawiera ogólne informacje o grze. Jest też kilka zmian konfiguracyjnych generatora świata, które są tu również opisane.",
+	"bingo.lobby.tutorial.game_info.general.heading": "Ogólne",
+	"bingo.lobby.tutorial.game_info.general": "Wszystkie zdobyte przedmioty muszą być dokładnie takie jak na karcie. Liczą się wszystkie przedmioty o tym samym wyglądzie (np. zaklęta książka nie wymaga konkretnego zaklęcia, ale czerwone łóżko już wymaga konkretnego koloru).",
+	"bingo.lobby.tutorial.game_info.world_gen.heading": "Zmiany Generatora Świata",
+	"bingo.lobby.tutorial.game_info.world_gen": "Minecraft Bingo ma pewne modyfikacje generatora świata.",
+	"bingo.lobby.tutorial.game_info.world_gen.tiny_biomes.heading": "Małe Biomy: ",
+	"bingo.lobby.tutorial.game_info.world_gen.tiny_biomes": "Biomy zostały zmniejszone dla ułatwienia eksploracji.\nZe względu na pewne ograniczenia, możesz zauważyć też inne różnice w ich układzie.",
+	"bingo.lobby.tutorial.game_info.world_gen.more_dungeons.heading": "Więcej Lochów: ",
+	"bingo.lobby.tutorial.game_info.world_gen.more_dungeons": "Aby uczynić lochy rozsądnymi do odnalezienia w szybkiej grze Bingo, ich liczba została zwiększona o współczynnik 12,5, dzięki czemu łatwo jest je znaleźć w każdej rundzie.",
+	"bingo.lobby.tutorial.game_info.world_gen.caves.heading": "Zmiany Jaskiń: ",
+	"bingo.lobby.tutorial.game_info.world_gen.caves": "Częstotliwość występowania jaskiń została nieznacznie zwiększona\n(z domyślnej 1/7 do 1/6).\n\nPonadto, biomy oceaniczne nie generują zalanych jaskiń na tej mapie.",
+	"bingo.lobby.tutorial.game_info.world_gen.no_mineshafts.heading": "Brak Kopalń: ",
+	"bingo.lobby.tutorial.game_info.world_gen.no_mineshafts": "Opuszczone kopalnie okazały się zbyt potężne, ponieważ dają łatwy dostęp do wielu skrzyń. Dlatego też w Bingo zostały one wyłączone.",
+	"bingo.lobby.tutorial.game_info.world_gen.ites.heading": "Brak -it'ów/-yt;ów: ",
+	"bingo.lobby.tutorial.game_info.world_gen.ites": "Wszystkie warianty kamieni, takie jak DiorYT, GranIT i AndezYT zostały całkowicie usunięte z generacji w tej wersji.\nZostało to zrobione w celu zmniejszenia bałaganu w ekwipunku.",
+	"bingo.lobby.tutorial.game_info.world_gen.ambient.heading": "Jasne otoczenie: ",
+	"bingo.lobby.tutorial.game_info.world_gen.ambient": "Włączony jest parametr wymiaru pozwalający widziać w ciemności, podobnie jak w Netherze.\nDziała to bez noktowizji, dzięki czemu nadal możesz w pewnym stopniu rozróżnić ciemność od światła",
```

Re-commenting it, cause previous review got outdated
```suggestion
	"bingo.lobby.tutorial.game_info.world_gen.ambient": "Włączony jest parametr wymiaru pozwalający widzieć w ciemności, podobnie jak w Netherze.\nDziała to bez noktowizji, dzięki czemu nadal możesz w pewnym stopniu rozróżnić ciemność od światła",
```

---

Don't know why... Couldn't commit it from web, so made it from desktop -> cffe04f 😅